CHANGELOG.md

```diff
@@ -1,26 +1,268 @@
+| Updated cart page to show loading state while fetching data from network                | **Update**  | [#2454][] |
+| Updated cache policy for Home page                                                      | **Update**  | [#2453][] |
+| Updated Venia's header to match the styleguide's app bar                                | **Update**  | [#2488][] |
+| Swapped usage of React Head to React Helmet Async                                       | **Update**  | [#2412][] |
+| Updated Service Worker to be more strict when doing catalog image checks                | **Update**  | [#2392][] |
+| Removed reference related to recently viewed items                                      | **Update**  | [#2387][] |
+| Changed the accordion section buttons to type `button`                                  | **Update**  | [#2335][] |
+| Changed the Venia loading spinner image to a simpler CSS spinner                        | **Update**  | [#2310][] |
+| Added check for the `isRequired` validation rule                                        | **Update**  | [#2303][] |
+| Implemented Shipping Information form for guest checkout                                | **Update**  | [#2285][] |
+| Added Shipping Methods form to Checkout page                                            | **Update**  | [#2280][] |
+| Made minor updates to the Order Summary feature                                         | **Update**  | [#2278][] |
+| Styled icons with CSS                                                                   | **Update**  | [#2272][] |
+| Added the Order Summary to the Checkout page                                            | **Update**  | [#2271][] |
+| Cleaned and made Cart and Checkout pages consistent                                     | **Update**  | [#2258][] |
+| Updated Checkout Page GraphQL query                                                     | **Update**  | [#2254][] |
+| Added category description (with PageBuilder support as well) in the category view      | **Update**  | [#2226][] |
+| Updated cms component to include meta data                                              | **Update**  | [#2159][] |
+| Changed the random swatch color to the actual color                                     | **Update**  | [#2151][] |
+| Fixed a button clipping bug                                                             | **Bugfix**  | [#2499][] |
+| Fixed a "Data Fetch Error" on product the page during offline mode                      | **Bugfix**  | [#2490][] |
+| Fixed a mini cart bug that displayed incorrect dimensions and spacing for swatch images | **Bugfix**  | [#2457][] |
+| Fixed dropdown arrows in Firefox                                                        | **Bugfix**  | [#2455][] |
+| Fixed price summary not updating when shipping info is updated                          | **Bugfix**  | [#2445][] |
+| Fixed css on the checkout page                                                          | **Bugfix**  | [#2416][] |
```

Comment:
"CSS" should be capitalized. Also see comment on line 77 for how to handle "checkout page".